extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -223,9 +224,27 @@ export const selectProjectLocation = localize('selectProjectLocation', "Select p
 export const ProjectParentDirectoryNotExistError = (location: string): string => { return localize('dataworkspace.projectParentDirectoryNotExistError', "The selected project location '{0}' does not exist or is not a directory.", location); };
 export const ProjectDirectoryAlreadyExistError = (projectName: string, location: string): string => { return localize('dataworkspace.projectDirectoryAlreadyExistError', "There is already a directory named '{0}' in the selected location: '{1}'.", projectName, location); };
 
+// Update Project From Database dialog strings
+
+export const updateProjectFromDatabaseDialogName = localize('updateProjectFromDatabaseDialogName', "Update project from database");
+export const updateProjectDialogOkButtonText = localize('updateProjectDialogOkButtonText', "Update");
```

Comment:
It would be great to get into the habit of having the loc IDs be as generic as possible - e.g. `update` for this one. Having the ID tied to a specific use just means it's more annoying to re-use